src/reason-parser/reason_parser.mly

```diff
@@ -1209,6 +1209,7 @@ let package_type_of_module_type pmty =
 %token SEMISEMI
 %token SHARP
 %token <string> SHARPOP
+%token SHARPOP_WITH_EQUAL
```

Comment:
can you rename this to `SHARPEQUAL`? `SHARPOP_WITH_EQUAL` conveys `# with = `, if you look at the other token names.

Reply:
definitely